.github/ISSUE_TEMPLATE/universal_questions.md

```diff
@@ -0,0 +1,7 @@
+**Question text:**
+
+**Support text or options:**
+
+**Question type **(drop down, radio, checklist, address, phone, ID, name, static text, text field):
```

Comment:
```suggestion
**Question type** (drop down, radio, checklist, address, phone, ID, name, static text, text field):
```

The space is in the wrong spot to make `Question type` bold

Reply:
It's a lesser known feature, but if you want an actual dropdown and/or multiple textboxes take a look at the [forms syntax](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#about-yaml-syntax-for-issue-forms).